runtime/doc/options.txt

```diff
@@ -3603,8 +3603,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 						*'fillchars'* *'fcs'*
 'fillchars' 'fcs'	string	(default "vert:|,fold:-,eob:~,lastline:@")
 			global or local to window |global-local|
-	Characters to fill the statuslines, vertical separators and special
-	lines in the window.
+	Characters to fill the statuslines, vertical separators, special
+	lines in the window and truncated text in the |popup-menu|.
```

Comment:
I don't think `|popup-menu|` is correct. `:h popup-menu` is completely unrelated to `'pummaxwidth'`.

Reply:
right update to `ins-completion-menu`